consensus/polybft/bridge_manager.go

```diff
@@ -229,6 +233,7 @@ func newBridgeManager(
 			startBlock:            runtimeConfig.GenesisConfig.Bridge[chainID].EventTrackerStartBlocks[stateSenderAddr],
 			trackerPollInterval:   runtimeConfig.GenesisConfig.BlockTrackerPollInterval.Duration,
 		},
+		state: state,
 	}
 
```

Comment:
We should dispose the event tracker when Close function is invoked. Therefore we need to have the reference to `eventTracker` in the `bridgeManager`.